src/document.h

```diff
@@ -830,6 +830,29 @@ struct Document {
         return nullptr;
     }
 
+    const wxChar* CopyImageToClipboard(Cell *cell) {
+        if (wxTheClipboard->Open()) {
+            #if defined(__WXMSW__) || defined(__WXGTK__)
+            if (!cell->text.image->png_data.empty()) {
+                wxCustomDataObject *pngimage = new wxCustomDataObject(
+                    #ifdef __WXMSW__
+                        wxDF_PNG
+                    #else
+                        wxDF_BITMAP
+                    #endif
+                );
+                pngimage->SetData(cell->text.image->png_data.size(), cell->text.image->png_data.data());
+                wxTheClipboard->SetData(pngimage);
+            } else
+            #endif
+            {
+                wxTheClipboard->SetData(new wxBitmapDataObject(cell->text.image->bm_orig));    
+            }
+            wxTheClipboard->Close();
+            }

```

Comment:
wrong indent here

Reply:
Thanks for taking your time! I corrected this in https://github.com/aardappel/treesheets/pull/301/commits/c84f3d994a415871736a6918c569f6e943116d66